controllers/vrg_volrep.go

```diff
@@ -647,16 +649,22 @@ func (v *VRGInstance) UploadPVAndPVCtoS3(s3ProfileName string, objectStore Objec
 		return err
 	}
 
+	v.log.Info("Uploaded PV to s3 profile", "PV", pv.Name, "s3 profile", s3ProfileName)
```

Comment:
Logging before and after an operation is important when the operation takes log of time, but these logs are debug logs. In INFO level logs it is good enough to before the operation. If the operation fails we should see an error log.

Reply:
These are INFO logs. Each block has error logging. 
2023-11-08T21:23:29.214+0530  INFO    controllers.VolumeReplicationGroup.vrginstance  About to upload PV to s3 profile        {"VolumeReplicationGroup": "envtest-ns-sdixm/vrg-sdixm", "rid": "d5b10778-cf83-48ca-897c-d810ed18bea0", "State": "primary", "PV": "pv0001", "s3 profile": "s3profile0"}
  2023-11-08T21:23:29.214+0530  INFO    controllers.VolumeReplicationGroup.vrginstance  Uploaded PV to s3 profile       {"VolumeReplicationGroup": "envtest-ns-sdixm/vrg-sdixm", "rid": "d5b10778-cf83-48ca-897c-d810ed18bea0", "State": "primary", "PV": "pv0001", "s3 profile": "s3profile0"}
  2023-11-08T21:23:29.214+0530  INFO    controllers.VolumeReplicationGroup.vrginstance  About to upload PVC to s3 profile       {"VolumeReplicationGroup": "envtest-ns-sdixm/vrg-sdixm", "rid": "d5b10778-cf83-48ca-897c-d810ed18bea0", "State": "primary", "PVC": "pvc-of-pv0001", "s3 profile": "s3profile0"}
  2023-11-08T21:23:29.214+0530  INFO    controllers.VolumeReplicationGroup.vrginstance  Uploaded PVC to s3 profile      {"VolumeReplicationGroup": "envtest-ns-sdixm/vrg-sdixm", "rid": "d5b10778-cf83-48ca-897c-d810ed18bea0", "State": "primary", "PVC": "pvc-of-pv0001", "s3 profile": "s3profile0"}
  2023-11-08T21:23:29.216+0530  INFO    pvcmap.VolumeReplicationGroup   Update event for PersistentVolumeClaim
  2023-11-08T21:23:29.216+0530  INFO    pvcmap.VolumeReplicationGroup   Reconciling due to VR Protection finalizer      {"pvc": "envtest-ns-sdixm/pvc-of-pv0001"}
  2023-11-08T21:23:29.216+0530  INFO    pvcmap.VolumeReplicationGroup   Found VolumeReplicationGroup with matching labels       {"pvc": "envtest-ns-sdixm/pvc-of-pv0001", "vrg": "vrg-sdixm", "labeled": ""}
  2023-11-08T21:23:29.216+0530  INFO    pvcmap.VolumeReplicationGroup   Found VolumeReplicationGroup with matching labels       {"pvc": "envtest-ns-sdixm/pvc-of-pv0001", "vrg": "vrg-sdixm", "labeled": ""}
  2023-11-08T21:23:29.218+0530  INFO    controllers.VolumeReplicationGroup.vrginstance  Done uploading PV/PVC cluster data to 1 of 1 S3 profile(s): [s3profile0]        {"VolumeReplicationGroup": "envtest-ns-sdixm/vrg-sdixm", "rid": "d5b10778-cf83-48ca-897c-d810ed18bea0", "State": "primary"}
  2023-11-08T21:23:29.218+0530  INFO    controllers.VolumeReplicationGroup.vrginstance  Successfully processed VolumeReplication for PersistentVolumeClaim      {"VolumeReplicationGroup": "envtest-ns-sdixm/vrg-sdixm", "rid": "d5b10778-cf83-48ca-897c-d810ed18bea0", "State": "primary", "pvc": "envtest-ns-sdixm/pvc-of-pv0001"}